tile.lua

```diff
@@ -188,13 +188,13 @@ function M.task(starts, ends, config, x1, y1, x2, y2)
 
     local age = api.clock.unix() - toot.created_at
     if age < 100 then
-        age = string.format("%ds", age)
+        age = string.format("%d Sekunden", age)
```

Comment:
These localization changes should be configurable. This plugin is used outside of german-speaking areas as well.